Index: nwchem-6.8/src/basis/bas_input.F
===================================================================
--- nwchem-6.8.orig/src/basis/bas_input.F
+++ nwchem-6.8/src/basis/bas_input.F
@@ -1253,12 +1253,11 @@ c
       noslash=.false.
       calls = calls + 1
       if (debug) write(luout,*)' calls:calls: ',calls
-      call util_nwchem_srcdir(compiled_name)
+c     call util_nwchem_srcdir(compiled_name)
 c
 c     add here basis/libraries bit
 c
-      compiled_name=compiled_name(1:inp_strlen(compiled_name))
-     $     //"/basis/libraries/"
+      compiled_name="/usr/share/nwchem/libraries/"
 *
 * order of precedence for choosing name
 * 1) value of NWCHEM_BASIS_LIBRARY environment variable
Index: nwchem-6.8/src/nwpw/libraryps/nwpw_libfile.F
===================================================================
--- nwchem-6.8.orig/src/nwpw/libraryps/nwpw_libfile.F
+++ nwchem-6.8/src/nwpw/libraryps/nwpw_libfile.F
@@ -118,8 +118,7 @@ c    &NWPW_LIBRARY
 c
 c     add here nwpw/libraryps bit
 c
-      libname=libname(1:inp_strlen(libname))
-     &     //"/nwpw/libraryps/"
+      libname="/usr/share/nwchem/libraryps/"
       if (util_find_dir(libname)) then
          goto 99
       else
