Index: nwchem-6.8/src/basis/bas_input.F
===================================================================
--- nwchem-6.8.orig/src/basis/bas_input.F
+++ nwchem-6.8/src/basis/bas_input.F
@@ -1295,13 +1295,14 @@ c
       noslash=.false.
       calls = calls + 1
       if (debug) write(luout,*)' calls:calls: ',calls
-      call util_nwchem_srcdir(compiled_name)
+c      call util_nwchem_srcdir(compiled_name)
 c
 c     add here basis/libraries bit
 c
       if(libname.ne.' ') then
-      compiled_name=compiled_name(1:inp_strlen(compiled_name))
-     $     //"/basis/"//libname(1:inp_strlen(libname))//"/"
+c      compiled_name=compiled_name(1:inp_strlen(compiled_name))
+c     $     //"/basis/"//libname(1:inp_strlen(libname))//"/"
+      compiled_name="/usr/share/nwchem/libraries/"
       else
       compiled_name=compiled_name(1:inp_strlen(compiled_name))
      $     //"/basis/libraries/"
Index: nwchem-6.8/src/nwpw/libraryps/nwpw_libfile.F
===================================================================
--- nwchem-6.8.orig/src/nwpw/libraryps/nwpw_libfile.F
+++ nwchem-6.8/src/nwpw/libraryps/nwpw_libfile.F
@@ -118,8 +118,7 @@ c    &NWPW_LIBRARY
 c
 c     add here nwpw/libraryps bit
 c
-      libname=libname(1:inp_strlen(libname))
-     &     //"/nwpw/libraryps/"
+      libname="/usr/share/nwchem/libraryps/"
       if (util_find_dir(libname)) then
          goto 99
       else
