--- nwchem/src/config/makefile.h.org	2020-03-05 19:04:43.814971579 -0800
+++ nwchem/src/config/makefile.h	2020-03-05 19:06:37.739577100 -0800
@@ -1297,14 +1297,14 @@
 endif
 
 
-ifeq ($(TARGET),$(findstring $(TARGET),LINUX CYGNUS CYGWIN))
+ifeq ($(TARGET),$(findstring $(TARGET),LINUX LINUX64 CYGNUS CYGWIN))
 #
 #
 # Linux or Cygwin under Windows running on an x86 using g77
 #
        NICE = nice -n 2
       SHELL := $(NICE) /bin/sh
-    CORE_SUBDIRS_EXTRA = blas lapack
+    CORE_SUBDIRS_EXTRA =
          CC = gcc
      RANLIB = ranlib
   MAKEFLAGS = -j 1 --no-print-directory
@@ -1328,7 +1328,7 @@
    endif
    endif
 
-         LINUXCPU = $(shell uname -m |\
+         LINUXCPU_ = $(shell uname -m |\
                  awk ' /sparc/ { print "sparc" }; /i*86/ { print "x86" };  /ppc*/ { print "ppc"};  /arm*/ { print "arm"}; /mips*/ { print "mips"} ' )
 
      GOTMINGW32= $(shell $(CC) -dM -E - </dev/null 2> /dev/null |grep MINGW32|cut -c21)
@@ -1349,7 +1349,7 @@
       COPTIONS   = -Wall
       COPTIMIZE  = -g -O2
       ifeq ($(_FC),gfortran)
-        FOPTIONS   = # -Wextra -Wunused  
+        FOPTIONS   = -g -O2 -ffast-math -fno-aggressive-loop-optimizations
         FOPTIMIZE  += -ffast-math -Wuninitialized
         DEFINES  += -DGFORTRAN
         GNUMAJOR=$(shell $(_FC) -dM -E - < /dev/null 2> /dev/null | grep __GNUC__ |cut -c18-)
@@ -1389,7 +1389,7 @@
     DEFINES += -DCYGWIN -DCYGNUS
   endif
   
-  _CPU = $(shell uname -m  )
+  _CPU_ = $(shell uname -m  )
 
       ifeq ($(FC),g77)
   _G77V33= $(shell g77 -v  2>&1|egrep spec|head -n 1|awk ' /3.3/  {print "Y"}')
@@ -1638,7 +1638,7 @@
 endif
 #EXTRA_LIBS +=-lefence # link against Electricfence
 
-CORE_LIBS += -lnwclapack $(BLASOPT) -lnwcblas
+#CORE_LIBS += -lnwclapack $(BLASOPT) -lnwcblas
 
 # end of Linux, Cygnus
 endif
@@ -1735,9 +1735,6 @@
       _CC=gcc
       endif
       FOPTIMIZE  = -O2 
-       ifeq ($(_CPU),aarch64)
-         DONTHAVEM64OPT=Y
-       endif
        ifeq ($(_CPU),mips64)
          DONTHAVEM64OPT=Y
          COPTIONS   = -mabi=64
@@ -2236,7 +2233,7 @@
           FOPTIONS +=  -ff2c -fno-second-underscore
         endif
         ifeq ($(GNU_GE_4_6),true) 
-          FOPTIMIZE +=  -mtune=native
+          FOPTIMIZE +=
           FOPTIONS += -finline-functions
         endif
 #        FVECTORIZE  += -ftree-vectorize -ftree-vectorizer-verbose=1
